web3/_utils/method_formatters.py

```diff
@@ -161,6 +164,12 @@ def type_aware_apply_formatters_to_dict(
     """
     Preserve ``AttributeDict`` types if original ``value`` was an ``AttributeDict``.
     """
+    # TODO: In v8, Use eth-utils 5.3.0 as lower pin where ``apply_formatters_to_dict``
```

Comment:
Do we want to pull CamelModel into web3 in this PR?

Reply:
I suppose we can. Should I add it to this PR?

Reply:
I'll leave that up to you!

Reply:
I added it in here 👍🏼

Reply:
OH we can't. `eth-account` can't inherit from ``CamelModel`` so we can't support it here yet since ``SignedSetCodeAuthorization`` is coming from eth-account.